datasync/datasync.go

```diff
@@ -367,3 +367,49 @@ func (d *Datasync) GetDatasyncTags(ctx context.Context, tArn string) ([]*datasyn
 
 	return out.Tags, err
 }
+
+// StartTaskExecution starts the execution and returns the taskexecution ARN
+func (d *Datasync) StartTaskExecution(ctx context.Context, taskArn string) (*datasync.StartTaskExecutionOutput, error) {
+	if taskArn == "" {
+		return nil, apierror.New(apierror.ErrBadRequest, "invalid input", nil)
+	}
+
+	log.Info("starting datasync task executions")
+
+	filters := &datasync.StartTaskExecutionInput{TaskArn: aws.String(taskArn)}
+
+	out, err := d.Service.StartTaskExecutionWithContext(ctx,
+		filters,
+		func(r *request.Request) {})
+
+	if err != nil {
+		return nil, ErrCode("failed to start task executions", err)
+	}
+
+	log.Debugf("listing datasync tasks execution output: %+v", out)
+
+	return out, nil
+}
+
+// StopTaskExecution Stop the execution and returns the taskexecution ARN
+func (d *Datasync) StopTaskExecution(ctx context.Context, taskArn string) (*datasync.CancelTaskExecutionOutput, error) {
+	if taskArn == "" {
+		return nil, apierror.New(apierror.ErrBadRequest, "invalid input", nil)
+	}
+
+	log.Info("Stopping datasync task executions")
+
+	filters := &datasync.CancelTaskExecutionInput{TaskExecutionArn: aws.String(taskArn)}
+
+	out, err := d.Service.CancelTaskExecutionWithContext(ctx,
+		filters,
+		func(r *request.Request) {})
+
```

Comment:
✂️  (no space between func call and err check)

Reply:
removed extra spacing.